blocks/hero/hero.js

```diff
@@ -1,7 +1,37 @@
 import { decorateIcons } from '../../scripts/lib-franklin.js';
 import { addClipboardInteraction } from '../../scripts/utils.js';
 
+function addBackLink(block) {
+  if (window.location.pathname.endsWith('/news-und-events/')) {
+    return;
+  }
+
+  const bl = document.createElement('div');
+  // The 2 elements are created in the 'backward' order to allow the hover over the link
+  // to change the color of the arrow. They are put in the correct position via the
+  // 'order' attribute in the CSS.
+  bl.innerHTML = `<a href="/de/semiconductor-manufacturing-technology/news-und-events/" class="back-link"> Zurück zu Informationen und Veranstaltungen</a>
+  <a href="#" id="back-link-icon">
+    <svg id="back-icon" focusable="false" xmlns:xlink="http://www.w3.org/1999/xlink"">
+      <use xlink:href="/icons/symbols-sprite.svg#svgsymbol-chevron-left"></use>
+    </svg>
+  </a>`;
```

Comment:
I think there's also a dark divider line that we need to have 

![image](https://user-images.githubusercontent.com/1191451/231212619-aa081932-cac7-4efc-bdd2-8e92c6cb1209.png)